overrides/cf.d.ts

```diff
@@ -289,97 +297,693 @@ interface RequestInitCfPropertiesImageMinify {
   verifiedBot: boolean;
-}
 
-interface IncomingRequestCfPropertiesTLSClientAuth {
-  certIssuerDNLegacy: string;
+  /**
+   * A boolean value that is true if the request originates from a
+   * Cloudflare-verified proxy service.
+   */
+  corporateProxy: boolean;
+
+  /**
+   * A boolean value that's true if the request matches [file extensions](https://developers.cloudflare.com/bots/reference/static-resources/) for many types of static resources.
+   */
+  staticResource: boolean;
+
+  /**
+   * A [JA3 Fingerprint](https://developers.cloudflare.com/bots/concepts/ja3-fingerprint/) to help profile specific SSL/TLS clients
+   * across different destination IPs, Ports, and X509 certificates.
+   *
+   * This field is only set on the Enterprise plan.
+   */
+  ja3Hash?: string;
+};
+
+/**
+ * Metadata about the request's TLS handshake
+ */
+type IncomingRequestCfPropertiesExportedAuthenticatorMetadata = {
+  /**
+   * The client's [`HELLO` message](https://www.rfc-editor.org/rfc/rfc5246#section-7.4.1.2), encoded in hexadecimal
+   *
+   * @example "44372ba35fa1270921d318f34c12f155dc87b682cf36a790cfaa3ba8737a1b5d"
+   */
+  clientHandshake: string;
+
+  /**
+   * The server's [`HELLO` message](https://www.rfc-editor.org/rfc/rfc5246#section-7.4.1.2), encoded in hexadecimal
+   *
+   * @example "44372ba35fa1270921d318f34c12f155dc87b682cf36a790cfaa3ba8737a1b5d"
+   */
+  serverHandshake: string;
+
+  /**
+   * The client's [`FINISHED` message](https://www.rfc-editor.org/rfc/rfc5246#section-7.4.9), encoded in hexadecimal
+   *
+   * @example "084ee802fe1348f688220e2a6040a05b2199a761f33cf753abb1b006792d3f8b"
+   */
+  clientFinished: string;
+
+  /**
+   * The server's [`FINISHED` message](https://www.rfc-editor.org/rfc/rfc5246#section-7.4.9), encoded in hexadecimal
+   *
+   * @example "084ee802fe1348f688220e2a6040a05b2199a761f33cf753abb1b006792d3f8b"
+   */
+  serverFinished: string;
+};
+
+/**
+ * Geographic data about the request's origin.
+ */
+type IncomingRequestCfPropertiesGeographicInformation =
+  | {
+      /* No geographic data was found for the incoming request. */
+    }
+  | {
+      /** The country code `"T1"` is used for requests originating on TOR  */
+      country: "T1";
+    }
+  | {
+      /**
+       * The [ISO 3166-1 Alpha 2](https://www.iso.org/iso-3166-country-codes.html) country code the request originated from.
+       *
+       * If your worker is [configured to accept TOR connections](https://support.cloudflare.com/hc/en-us/articles/203306930-Understanding-Cloudflare-Tor-support-and-Onion-Routing), this may also be `"T1"`, indicating a request that originated over TOR.
+       *
+       * If Cloudflare is unable to determine where the request originated this property is omitted.
+       *
+       * @example "GB"
+       */
+      country: Iso3166Alpha2Code;
+
+      /**
+       * If present, this property indicates that the request originated in the EU
+       *
+       * @example "1"
+       */
+      isEUCountry?: "1";
```

Comment:
It's the string "1"? I feel like a boolean would make more sense here... (not that you can control that, of course)